nanome/_internal/_ui/_button.py

```diff
@@ -32,6 +32,9 @@ def _register_pressed_callback(self, func):
         self._pressed_callback = func
 
     def _register_hover_callback(self, func):
+        nanome._internal._network._ProcessNetwork._instance._send(
+            nanome._internal._network._commands._callbacks._Messages.hook_ui_callback,
+            (nanome._internal._network._commands._serialization._UIHook.Type.button_hover, self._content_id))
         self._hover_callback = func
```

Comment:
In `_copy_values_deep`, `_hover_callback` is copied to the second button, but this doesn't register the hook, does it?

Reply:
Nice catch